internal/lightcone/abundance/warmthshortenscoldnights/warmthshortenscoldnights.go

```diff
@@ -0,0 +1,67 @@
+package warmthshortenscoldnights
+
+import (
+	"github.com/simimpact/srsim/pkg/engine"
+	"github.com/simimpact/srsim/pkg/engine/equip/lightcone"
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/engine/modifier"
+	"github.com/simimpact/srsim/pkg/engine/prop"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	WarmthShortensColdNightsHPBuff     key.Modifier = "warmth_shortens_cold_nights_hp_buff"
+	WarmthShortensColdNightsHealCheck  key.Modifier = "warmth_shortens_cold_nights_heal_check"
+)
+
+func init() {
+	lightcone.Register(key.WarmthShortensColdNights, lightcone.Config{
+		CreatePassive: Create,
+		Rarity:        4,
+		Path:          model.Path_ABUNDANCE,
+		Promotions:    promotions,
+	})
+
+	//hp buff goes here
+	modifier.Register(WarmthShortensColdNightsHPBuff, modifier.Config{
+		StatusType: model.StatusType_STATUS_BUFF,
```

Comment:
The "Increases the wearer's Max HP by X%" part of this is not registered as a `BUFF` in the game. The game is pretty arbitrary about what is/is not a buff. Typically static stuff in light cones like this will never be a `BUFF`